### PR TITLE
feat: make masterchain gas price configurable

### PR DESCRIPTION
**Pull request by sokel**
This option now usefull for `deposit` and gatekeeper `withdraw`;
If option not defined gas price is determined by the latest blocks median gas price.

### CODE REVIEW

File: blockchain/config.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ type Config struct {
 	SidechainEndpoint    url.URL
 	ContractRegistryAddr common.Address
 	BlocksBatchSize      uint64
+	MasterchainGasPrice  int64
```
Comment (3Hren) [inline blockchain/config.go]: What if it is negative?

File: etc/node.yaml [hunk 2]
```diff
@@ -51,3 +51,9 @@ ethereum:
   pass_phrase: "any"
 
 metrics_listen_addr: "127.0.0.1:14003"
+
+
+blockchain:
+  # gas price for masterchain transactions, in wei
+  # default is 20 Gwei
+  masterchain_gas_price: 20000000000
```
Comment (3Hren) [inline etc/node.yaml]: Wouldn't it better to write directly "20Gwei" instead, like we do in duration fields?

File: blockchain/options.go [hunk 3]
```diff
@@ -63,14 +63,14 @@ type options struct {
...
-			gasPrice:           defaultMasterchainGasPrice,
+			gasPrice:           big.NewInt(0).SetUint64(defaultMasterchainGasPrice),
 			gasLimit:           defaultMasterchainGasLimit,
 			endpoint:           defaultMasterchainEndpoint,
 			logParsePeriod:     defaultLogParsePeriod,
 			blockConfirmations: defaultBlockConfirmations,
 		},
 		sidechain: &chainOpts{
-			gasPrice:           defaultSidechainGasPrice,
+			gasPrice:           big.NewInt(0).SetUint64(defaultSidechainGasPrice),
```
Comment (sshaman1101) [inline blockchain/options.go]: big.NewInt(defaultSidechainGasPrice)

Comment (sokel) [reply blockchain/options.go]: fixed

**Pull request merged**